server/datastore/mysql/statistics.go

```diff
@@ -32,6 +32,30 @@ func (ds *Datastore) ShouldSendStatistics(ctx context.Context, frequency time.Du
 		if err != nil {
 			return ctxerr.Wrap(ctx, err, "amount users")
 		}
+		amountSoftwares, err := amountSoftwaresDB(ctx, ds.writer(ctx))
```

Comment:
@getvictor Should we change this whole cron to use `ds.reader(ctx)`? I don't see a reason to use the `ds.writer(ctx)`...

Reply:
Yes, the cron should use `ds.reader(ctx)`

Reply:
@lucasmrod @getvictor should this not be a separate PR ?

Reply:
Am ok with making the ds.writer -> ds.reader changes on a separate PR (we are already using ds.writer everywhere on this cron job)